serverless/jobs/reference-content/jobs-limitations.mdx

```diff
@@ -0,0 +1,45 @@
+---
+meta:
+  title: Jobs limitations
+  description: This section contains usage limits that apply when using Serverless Jobs.
+content:
+  h1: Jobs limitations
+  paragraph: This section contains usage limits that apply when using Serverless Jobs.
+tags: Jobs limitations serverless
+dates:
+  validation: 2024-03-28
+  posted: 2021-03-28
+categories:
+  - serverless
+  - Jobs
+---
+
+This section contains usage limits that apply when using Serverless Jobs.
+
+## Jobs resource limitations
+
+| Resource                      | Criteria   | Limit      | Scope        |
+|-------------------------------|----------- |------------|--------------|
+| Number of job definitions max | Max number | 1000       | Organization |
```

Comment:
I would maybe get rid of the "max"es in the resource column, as I don't think they're necessary given that stated in the criteria column (and would be easier to read without the repeated "max")